### ISSUE TITLE
Linux Mint network manager prevents dory to launch

### ISSUE THREAD

**Issue by rmNyro** [FreedomBen/dory#12]
Hi,

[Apparently](https://unix.stackexchange.com/a/304129) dnsmasq is used as part of the network manager in Mint. I don't know what to do with that knowledge but I found the answer while posting the issue and thought you may be interested in at least documenting it.

You may add a script to automate the "problem" resolution such as:
```sh
sudo sed -i 's/^dns=dnsmasq/#&/' /etc/NetworkManager/NetworkManager.conf \
&& sudo service network-manager restart \
&& sudo service networking restart
```

I'm not sure it is the best solution but so far I got no trouble (even a connectivity improvement?) except that it doesn't necessarily reconnect correctly (apparently?).

I hope that was helpful,

cheers,

**Comment by FreedomBen** [comment]
Yes it's helpful, thank you!  I'll think this over, and probably download a Mint iso and play around with it
